clcache.py

```diff
@@ -1371,24 +1379,24 @@ def processCacheHit(cache, objectFile, cachekey):
         return 0, cachedArtifacts.stdout, cachedArtifacts.stderr, False
 
 
-def createManifest(manifestHash, includePaths):
-    baseDir = normalizeBaseDir(os.environ.get('CLCACHE_BASEDIR'))
```

Comment:
Fair enough. For what it's worth, I was just curious, I didn't actually benchmark it. :-)